### PR TITLE
TODO more options

### PR DESCRIPTION
**Pull request by Momjo**

### CODE REVIEW

**Comment by AmirSavand** [comment]
What options? or features?

**Comment by AmirSavand** [comment]
For date picker: https://jqueryui.com/datepicker/

**Comment by Momjo** [comment]
for now only the description and Date, that when the to do need to be done

File: TO-DO/js/TODO-js.js [hunk 1]
```diff
@@ -66,3 +66,8 @@ function removeTodo(element) {
   // Remove the button
   $(element).remove();
 }
+
+function newTodoOptions(){
+  $('#moreOptions').addClass('shadow');
+  $('#moreOptions').slideToggle();
+}
```
Comment (AmirSavand) [inline TO-DO/js/TODO-js.js]: Add new line at the end of the file.

File: TO-DO/css/TODO-css.css [hunk 2]
```diff
@@ -20,3 +21,9 @@ input {
 #done .todo:first-child {
     margin: 0 !important;
 }
+
+/* TODO Options */
+
+#moreOptions{
+    display: none;
+}
```
Comment (AmirSavand) [inline TO-DO/css/TODO-css.css]: Add new line at the end of the file.

File: TO-DO/TODO-html.html [hunk 3]
```diff
@@ -70,4 +95,4 @@ <h2>Already Done</h2>
     </div>
 </body>
 
-</html>
+</html>
```
Comment (AmirSavand) [inline TO-DO/TODO-html.html]: Add new line at the end of the file.

File: TO-DO/js/TODO-js.js (hunk 1 above)
Comment (AmirSavand) [inline TO-DO/js/TODO-js.js]: I need a screenshot.

**Review by AmirSavand — CHANGES_REQUESTED** [review]

**Comment by Momjo** [comment]
screen shot 2018-06-01 at 13 04 06

screen shot 2018-06-01 at 13 04 13

**Review by AmirSavand — APPROVED** [review]

**Pull request merged**